src/main/java/bio/terra/dao/StudyDao.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.dao;
+
+import bio.terra.configuration.DataRepoJdbcConfiguration;
+import bio.terra.metadata.Study;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.UUID;
+
+@Repository
+public class StudyDao extends MetaDao<Study> {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    private TableDao tableDao;
+
+    @Autowired
+    private RelationshipDao relationshipDao;
+
+
+    @Autowired
+    public StudyDao(DataRepoJdbcConfiguration jdbcConfiguration) {
+        jdbcTemplate = new NamedParameterJdbcTemplate(jdbcConfiguration.getDataSource());
```

Comment:
I think we should make the jdbcTemplate a Bean and auto-wire that instead of having each DAO mint its own.
Not a big deal, but some savings in memory footprint.

Reply:
done